org.opent2t.sample.thermostat.superpopular/com.wink.thermostat/js/tests/test.js

```diff
@@ -0,0 +1,65 @@
+/*jshint esversion: 6 */
+var opent2t = require('opent2t');
+var ava = require('ava');
+var config = require('./testconfig');
+var delay = require('delay')
+
+// This boilerplate code is copied from transpiled typescript AVA test code, which enables usage of 'yield' keyword. 
+var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, P, generator) {

```

Comment:
This is .js file. I felt this was looking clean and we really want us to use a test runner like ava, for all its benefits. when we setup lab, we know how many total test cases and how many passed etc.. May be we can simplify this by removing this boilerplate code and fixing it.

---
In reply to: [75198452](https://github.com/openT2T/translators/pull/53#discussion_r75198452) [](ancestors = 75198452)